jupyter-notebooks/analysis-ready-data/ard_2_use_case_1.ipynb

```diff
@@ -152,92 +197,92 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 35,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# create an API Request from the search specifications\n",
+    "# Create an API Request from the search specifications\n",
     "\n",
     "item_type = ['PSScene']\n",
     "\n",
     "geom_filter = data_filter.geometry_filter(test_aoi_geom)\n",
     "clear_percent_filter = data_filter.range_filter('clear_percent', 90)\n",
-    "date_range_filter = data_filter.date_range_filter(\"acquired\", datetime(month=4, day=1, year=2019), datetime(month=5, day=1, year=2019))\n",
+    "date_range_filter = data_filter.date_range_filter(\"acquired\", datetime(\n",
+    "    month=4, day=1, year=2019), datetime(month=5, day=1, year=2019))\n",
     "cloud_cover_filter = data_filter.range_filter('cloud_cover', None, 0.1)\n",
     "\n",
-    "combined_filter = data_filter.and_filter([geom_filter, clear_percent_filter, date_range_filter])\n",
+    "combined_filter = data_filter.and_filter(\n",
+    "    [geom_filter, clear_percent_filter, date_range_filter])\n",
     "\n",
     "async with Session() as sess:\n",
     "    cl = DataClient(sess)\n",
```

Comment:
Make sure the new SDK syntax is reflected here:
 async with Session() as sess:
    cl = sess.client('data')

This syntax applies for data, orders, etc.

Also, as you run these SDK code snippets, make sure your working environment is using the most recent version of the SDK.